behind-the-veil-siteroot/imports/ui/components/pages/artist/SpecificArtistPage.jsx

```diff
@@ -95,7 +96,7 @@ const SpecificArtistPage = () => {
                                             serviceLocation={userData.profile.artistServiceLocation}
                                             serviceRadius={userData.profile.artistServiceRadius} />,
                             <ArtistGalleryTab key={"gallery"} username={artistUsername} external={true} />,
-                            <span key={4}>Reviews</span>,
+                            <ArtistReviewsTab key={4} username={artistUsername}>Reviews</ArtistReviewsTab>,
```

Comment:
Genuine curious but do you know why the key for this is 4 and everything else is a string?